README.md

```diff
@@ -86,7 +162,7 @@ sudo systemctl start winetricks_update.timer
 ```
 Or using the more tradional Unix crontab...
 ```
-sudo cp "/usr/local/bin/update_winetricks.sh" "/etc/cron.weekly/"
+sudo ln "/usr/bin/update_winetricks.sh" "/etc/cron.weekly/update_winetricks.sh"
 ```
```

Comment:
Suggestion: considering how much simpler this method is, I would list it first.

Reply:
Could call... I'll put a little warning for **systemd** lovers there...